rules.yaml

```diff
@@ -280,6 +280,7 @@
 - { name: openxcom,                    verpat: "2016.*",                                     ignorever: true } # git snapshots
 - { name: p5-catalyst-plugin-authentication, ver: "0.100230",                                ignorever: true }
 - { name: p5-device-serialport,        ver: "1.040000",                                      ignorever: true }
+- { name: tt-rss,                      verpat: "r|201[67]",                                  ignorever: true } # Kind of snapshot release
```

Comment:
Pattern is broken (needs ```.*```s). And I don't think we can ignore it cause

> Tiny Tiny RSS uses a rolling release model based on git master branch which is considered stable

Let's postpone this until we can change versions with rules, so we could transform dates to a single format and ignore other versions.